src/webgpu/api/validation/capability_checks/limits/maxTextureDimension1D.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { kLimitBaseParams, makeLimitTestGroup } from './limit_utils.js';
+
+const limit = 'maxTextureDimension1D';
+export const { g, description } = makeLimitTestGroup(limit);
+
+g.test('at_over')
```

Comment:
Sorry for just thinking of this - the name of the test should probably include which entry point it's testing, something like `'createTexture,at_over'`. Some limits are validated in multiple places.

Reply:
done